documentation/src/main/asciidoc/user_guide/streams.adoc

```diff
@@ -118,18 +118,16 @@ This relies on the spec to pick the most specific method as defined link:https:/
 
 In our previous example we used a `Collector` to collect all the results into a `Map`.
 Unfortunately the link:{jdkdocroot}/java/util/stream/Collectors.html[Collectors]
-class doesn't produce Serializable instances.  Thus if you need to use these, you can use the newly provided
-link:{javadocroot}/org/infinispan/stream/CacheCollectors.html[CacheCollectors]
```

Comment:
I meant more that we should keep this paragraph in here in some way. We can reduce it, but just so people know about the `CacheCollectors` in case if they don't use the `CacheStream` interface.

Reply:
Hmmm... I've rebased and fixed other bits. Don't have time today to have an in depth look into this. Back on Friday, feel free to send me a PR if you have a clear idea and have time.